docker/bin/cluster_up.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+set -eu
+
+DOCKER_BIN="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+source ${DOCKER_BIN}/../.env
+
+# Wait for http service listener to come up and start serving
+# $1 http service name
+# $2 http service address
+await_http() {
+    local exit_code
+    local attempt=0
+
+    until [[ $(curl ${2}) ]] || [[ ${attempt} -gt 5 ]]; do
```

Comment:
I dont think you should call curl in a subshell, that's to be used when collecting output.